src/rules/noDupe.js

```diff
@@ -0,0 +1,32 @@
+import forEach from 'lodash/forEach';
```

Comment:
Please import the entire lodash bundle.

We are using `babel-plugin-lodash` to rewire dependencies in a build step.